ui/app/pages/unlock-page/unlock-page.component.js

```diff
@@ -10,6 +10,7 @@ import { DEFAULT_ROUTE } from '../../helpers/constants/routes'
 export default class UnlockPage extends Component {
   static contextTypes = {
     metricsEvent: PropTypes.func,
+    mixPanelTrack: PropTypes.func,
```

Comment:
It looks like the context property provided by the legacy mixpanel provider is `mixPanelEvent`, not `mixPanelTrack`